test/translate_c.zig

```diff
@@ -3917,4 +3917,19 @@ pub fn addCases(cases: *tests.TranslateCContext) void {
         \\    };
         \\}
     });
+
+    cases.add("void pointer subtraction gives a result in bytes",
```

Comment:
This test case should be added in `test/cases/translate_c` (see comment at top of file)

Reply:
Oops, good point, thanks!